subcommands/keys/tuf_updates_rotate_online_key.go

```diff
@@ -0,0 +1,94 @@
+package keys
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"golang.org/x/exp/slices"
+
+	"github.com/foundriesio/fioctl/subcommands"
+)
+
+func init() {
+	rotate := &cobra.Command{
+		Use:   "rotate-online-key --role targets|snapshot|timestamp [--txid=<txid>]",
+		Short: "Stage rotation of the online TUF signing key for the Factory",
+		Long: `Stage rotation of the online TUF signing key for the Factory.
+
+The new online signing key will be used in both CI and production TUF root.
+
+When you rotate the TUF online signing key:
+- if there are CI or production targets in your factory, they are re-signed using the new key.
+- if there is an active wave in your factory, the TUF online key rotation is not allowed.
+- the new wave cannot be created until you apply the online keys rotation.
+
+When you apply the online key rotation, these features are temporarily disabled until it succeeds:
+- new CI targets upload (including the targets upload during CI builds).
+- automatic re-signing of expired TUF roles using online keys (both CI and production targets).
+
+Please, plan the TUF online key rotation timing accurately in order to prevent service disruption.`,
```

Comment:
You may want to elaborate on what you mean here. I sort of read this, got scared and didn't know of anything actionable I could do.

Reply:
I think I should just remove the last sentense.

It is the summary for the above two paragraphs: I wanted to put something string enough to let people re-read those two paragraphs.

But if this instead creates the confusion, it is definitely not the desired effect.

Reply:
The worst thing that could happen during online key rotation is a failed CI build. That's not bad enough to scare the user.

Reply:
Removed the last paragraph in the force-push